### PR TITLE
Asher to prevalence

### PR DESCRIPTION
**Pull request by asherps**
Added a to_latent_prevalence method to convert from an IncidenceRate to a latent prevalence. Defined a YearsOfInfections object to make this possible. If we know the number of yearly infections and the approximate number of years these infections have been occurring for, we would then be able to convert the annual incidence to a latent prevalence.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "[python]": {
+        "editor.defaultFormatter": "ms-python.black-formatter"
+    },
+    "python.formatting.provider": "none"
+}
```
Comment (jeffkaufman) [inline .vscode/settings.json:6]: This doesn't belong in this PR

File: pathogen_properties.py [hunk 2]
```diff
@@ -261,6 +266,15 @@ def to_prevalence(self, shedding_duration: SheddingDuration) -> Prevalence:
             active=Active.ACTIVE,
         )
 
+    def to_latent_prevalence(self, years: YearsOfInfections) -> Prevalence:
+        return Prevalence(
+            infections_per_100k=self.annual_infections_per_100k
+            * years.years
+            / 365,
```
Comment (jeffkaufman) [inline pathogen_properties.py:273]: Why this factor of 365?

**Review by jeffkaufman — CHANGES_REQUESTED** [review]
Can you say more about how you intend to use this?  I'm nervous about this as a concept because it assumes (a) the incidence rate has been constant over the time period and (b) a constant population (no births, deaths, or migrants).

Happy to call (in person would be better but I'm wfh) if that would be easier?

**Comment by simonleandergrimm** [comment]
I will review this pull request as soon as I/we refocus on vzv, but I don't want it to remain in the PR queue up until then.

**Pull request closed without merging**